### DISCUSSION TITLE
Change tux logo to linux logo

### DISCUSSION THREAD

**Discussion by toffeehpa** [PlagaMedicum/PlagueSur-icon-theme#3] [Ideas]
As good as the old linux logo was, in the days of minimalism, the old linux logo doesn't match modern designs at all. I suggest you another option that could very well make your icon pack much more individual. 
[new-linux-logo.tar.gz](https://github.com/PlagaMedicum/PlagueSur-icon-theme/files/11395220/new-linux-logo.tar.gz)
Also, many will probably agree with me, but I think the QuickTime logo on the .mp4 files and so on is not good. All in all, I'd like to talk to you personally about this topic

**Comment by PlagaMedicum** [comment]
Hey! Thanks for your feedback!
- I agree that the present logo is not the best in terms of minimalism and recognizability at low resolutions, so it needs to be replaced if possible. I'll consider the option you suggested when I'll have time to.
- If you have some suggestions for the QuickTime logo, please send it. If I'll find something more distinguishable, I'll adapt it.
